managed-services/transactional-email/concepts.mdx

```diff
@@ -11,74 +11,143 @@ categories:
 ---
 
 <Concept opened>
-
 ## Deliverability
 
 Deliverability is the ability for an email to arrive in a recipient’s email inbox.
-
 </Concept>
 
 <Concept opened>
-
 ## DKIM record
 
 A Domain Keys Identified Mail (DKIM) record is an email security standard. It provides you with an encryption key and a digital signature that allows you to verify whether an email was altered.
-
 </Concept>
 
 <Concept>
-
 ## DNS propagation
 
 DNS propagation is the time a DNS change takes to be updated across the Internet.
+</Concept>
+
+<Concept>
+## Hard bounce
+
+Hard bounce occurs when an email is sent to an invalid email address or an address that does not exist: unlike [soft bounce](#soft-bounce), which is a temporary error, hard bounce is a permanent error.
 
+In this case, most emailing solutions will blacklist these email addresses.
 </Concept>
 
 <Concept>
+## IP warming
 
-## IP reputation
+IP warming is the process of gradually increasing the volume of emails sent through a new IP or an API that has not been used in a while.
 
-IP reputation measures the trustworthiness of an IP address and the amount of unwanted requests it sends. An IP address that sends authentic, spam-free emails, is assigned a good IP reputation. An IP address that sends high amounts of spam or viruses is assigned a bad IP reputation. IP reputation allows you to reject requests that are coming from an IP address with a bad reputation.
+The process of warming up an IP address consists in gradually sending emails from it, starting with small volumes and increasing the volume over a period of days weeks or months, according to a set schedule.
+
+Internet service providers are wary of emails sent from new IP addresses with no reputation, so warming up an IP allows for better email deliverability and thus, better reputation.
 
 </Concept>
 
 <Concept>
+## IP reputation
 
+IP reputation measures the trustworthiness of an IP address and the amount of unwanted requests it sends. An IP address that sends authentic, spam-free emails, is assigned a good IP reputation. An IP address that sends high amounts of spam or viruses is assigned a bad IP reputation. IP reputation allows you to reject requests that are coming from an IP address with a bad reputation.
```

Comment:
```suggestion
IP reputation measures the trustworthiness of an IP address and the amount of unwanted requests it sends. An IP address that sends authentic, spam-free emails, earns a good IP reputation. An IP address that sends high amounts of spam or viruses earns a bad IP reputation. IP reputation allows you to reject requests that are coming from an IP address with a bad reputation.
```

Reply:
I made these suggestions because to my ear it sounds a bit weird to say that a reputation can be "assigned". Alternatively you could replace "earns" with "gets" ?